avocado/utils/network/interfaces.py

```diff
@@ -75,6 +75,11 @@ def _get_interface_details(self, version=4):
             log.error(msg)
             raise NWException(msg)
 
+    def _get_bondinterface_details(self, version=4):
```

Comment:
The `version` parameter is not used here.

Reply:
@vaishnavibhat if my comment was accurate, and `version` was not used on that version, my suggestion was in the lines of removing the parameter.  I see that the current version *does* use `version`, so I'd go with a better default such as you had before (and matches `getipaddrs`).

Reply:
@vaishnavibhat I apologize if my previous comment was mistaken and led to rework.

Reply:
IIUC, this has still some work to do here, since version is not being used on this method.